README.md

```diff
@@ -3,16 +3,17 @@
 [![Coverage](https://img.shields.io/codecov/c/github/luna/basegl?label=Coverage&labelColor=2c3239)](https://codecov.io/gh/luna/basegl/branch/master) 
 ![Stability](https://img.shields.io/static/v1?label=Stability&message=Unstable&color=d52229&labelColor=2c3239)
 
-# BaseGL
+# Enso Studio
 
-BaseGL is a blazing fast 2D vector rendering engine with a rich set of
+Enso Studio is an IDE for hybrid visual and textual functional programming.
+
+Its code uses BaseGL, a blazing fast 2D vector rendering engine with a rich set of
```

Comment:
I believe this is not the best way of changing it for several reasons:

- The fact that Enso Studio uses Basegl is not important to anyone reading this documentation. Sure we can mention it, but then writing a lot about BaseGL does not make sense. Telling later "This repository is a work in progress. Please refer to BaseGL 1.0 repository for more information:" makes even less sense in such context.
- The documentation should be crisp and concise.

I understand why you want this change, but it should be done different way around:

1. All description of Basegl should be moved to `lib/basegl/README.md`. Later we should refactor different libraries to `lib/basegl`, just as we have `lib/ide`.
2. The top-level `README.md` should describe only Enso Studio and how we should build it (like it describes it now). It can mention in 1 sentence that the GUI bases on BaseGL with a link to `lib/basegl.README.md`.

Let's always think about all the libs as separate things which we will refactor in the future to separate repos.